CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
+   *  Fixed a bug in Azure CNI Pod Subnet Static Block Allocation mode with Cilium which caused incorrect iptables rules, leading to pod connectivity failures to DNS and IMDS.
+
+
+* Behavior Changes:
+   * Starting in the Karpenter v0.6 image, Node Auto Provisioning (NAP) will bootstrap nodes from Scriptless to AgentBaker using custom data provided by Node Provisioning Service (NPS) and improve the stability of NAP.
+
+* Component Updates:
+   * Beginning in Kubernetes 1.32, AKS uses CNS to install Azure CNI NodeSubnet by default, meaning users will now see the CNS daemonset in their environments with no changes to functionality or required action.
+   * Node Auto Provisioning has been upgraded to use Karpenter [v0.7.2](https://github.com/Azure/karpenter-provider-azure/releases/tag/v0.7.2).
+   * Updated LTS patch version 1.27.102 for Command Injection affecting Windows nodes to address [CVE-2024-9042](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2024-9042).
+   * Updated the Retina basic image to v0.0.25 for Linux and Windows to address [CVE-2025-23047](https://nvd.nist.gov/vuln/detail/CVE-2025-23047) and [CVE-2024-45338](https://nvd.nist.gov/vuln/detail/CVE-2024-45338).
+   * Updated the cost-analysis-agent image from v0.0.20 to v0.0.21. Upgrades the following dependencies in cost-analysis-agent to fix [CVE-2024-45341](https://nvd.nist.gov/vuln/detail/CVE-2024-45341) and [CVE-2024-45336](https://nvd.nist.gov/vuln/detail/CVE-2024-45336):
+      * github.com/Azure/azure-sdk-for-go/sdk/azcore v1.15.0 to v1.17.0
+      * github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.8.0 to v1.8.1
+      * github.com/prometheus/common v0.60.0 to v0.62.0
+      * github.com/samber/lo v1.47.0 to v1.49.1
+      * github.com/stretchr/testify v1.9.0 to v1.10.0
+   * addon-resizer image v1.8.23-2 has been built using 
+    the Dalec framework, tested and published to MCR under [oss/v2 path](mcr.microsoft.com/oss/v2/kubernetes/autoscaler/addon-resizer:v1.8.23-2). All AKS clusters will use v1.8.23-2 addon-resizer image version.
+   * Updated Azure File CSI driver liveness probe to v2.15.0 and CSI node driver registrar to v2.13.0 on AKS 1.31 and 1.32.
+   * cluster-proportional-autoscaler has been upgraded to v1.9.0-1.
+   * Extend the AKS security patch release notes in release tracker to include a package comparison with the current - 1 AKS Ubuntu base image.
+   * AKS Azure Linux v2 image has been updated to [202502.09.0](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSAzureLinux/gen2/202502.09.0.txt).
+   * AKS Ubuntu 22.04 node image has been updated to [202502.09.0](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSUbuntu/gen2/2204containerd/202502.09.0.txt).
+   * AKS Windows Server 2019 image has been updated to [17763.6775.250117](https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/release-notes/AKSWindows/2019-containerd/17763.6775.250117.txt).
```

Comment:
I would suggest removing all the Linux, Windows release info as they are completely orthogonal to the AKS Release train as each has its own toggle in different cadence. Pls put the OS link from release tracker so ppl can self serve - https://releases.aks.azure.com/webpage/index.html

Reply:
let's discuss this internally as it's a departure from our existing process. for this release, I suggest we keep them